fmriprep/workflows/bold/registration.py

```diff
@@ -379,13 +396,16 @@ def init_bbreg_wf(
         (bbregister, transforms, [('out_lta_file', 'in1')]),
     ])  # fmt:skip
 
+    if use_t2w:
+        workflow.connect(fssource, 'T2', bbregister, 'intermediate_file')
+
```

Comment:
Does not do anything with `--init-reg` set, and nothing useful with `--init-header` set.

```suggestion
```